docs/docker.md

```diff
@@ -0,0 +1,58 @@
+# How to run geOrchestra on Docker
+
+## Using pre-built images
+
+**First step**
+
+Download the `docker-compose.yml` file:
+```
+wget https://raw.githubusercontent.com/georchestra/georchestra/master/docker-compose.yml
+```
+This file describes:
+ * which images / webapps will run,
+ * how they are linked together,
+ * where the configuration and data volumes are
+
+Feel free to comment out the apps you do not need.
+
+
+**Second step**
+
+Create a configuration directory on your host machine:
+```
+git clone -b docker-master https://github.com/georchestra/datadir.git
+sudo mv datadir /etc/georchestra
+```
+
+Optional: adjust the configuration in /etc/georchestra according to your needs.
+
+**Third step**
+
+Run geOrchestra with
+```
+docker-compose up
+```
+
+Open [http://localhost:8080/header/](http://localhost:8080/header/) in your browser
+
+
+## Building your own images
```

Comment:
I think this part should be moved to `docker/README` page. Because it targets skilled users ?